mungegithub/mungers/assign-unassign-handler.go

```diff
@@ -87,9 +88,18 @@ func (h AssignUnassignHandler) Munge(obj *github.MungeObject) {
 
 	toAssign, toUnassign := h.getAssigneesAndUnassignees(obj, comments, fileList, potentialOwners)
 	for _, username := range toAssign.List() {
-		obj.AssignPR(username)
+		err := obj.AssignPR(username)
+		if err != nil {
+			glog.Errorf("Could not assign %v to PR %v because %v", username, obj.Issue.Number, err)
+		}
+	}
+	for _, username := range toUnassign.List() {
+		err := obj.UnassignPR(username)
+		if err != nil {
+			glog.Errorf("Could not unassign %v from PR %v because %v", username, obj.Issue.Number, err)
+		}
 	}
-	obj.UnassignPR(toUnassign.List()...)
```

Comment:
As I was re implementing it as you described, I realize another downside.  If assigns and unassigns are aggregated, then if someone not on the org/project comments `/assign` or `/unassign` and github's api returns an error, they effectively prevent everyone else from being assigned/unassigned.